synthtool/gcp/templates/python_library/noxfile.py.j2

```diff
@@ -166,7 +166,7 @@ def docs(session):
     """Build the docs for this library."""
 
     session.install('-e', '.')
-    session.install('sphinx<3.0.0', 'alabaster', 'recommonmark')
+    session.install('sphinx', 'alabaster', 'recommonmark')
```

Comment:
@crwilcox   IIRC, you needed a specific version of sphinx.  Will this break you?

Reply:
We made this change here: https://github.com/googleapis/synthtool/pull/576 and the docs build passed successfully, so we wanted to make this change more broadly.

Reply:
@kolea2 is correct. A contributor made this change to bigtable specifically. And it seems @frankyn has had the same experience for storage specifically. This change undoes a mitigation while we were incompatible with newer versions of sphinx.